test/jasmine/tests/config_test.js

```diff
@@ -760,4 +760,61 @@ describe('config argument', function() {
             .then(done);
         });
     });
+
+    describe('scrollZoom:', function() {
+        var gd;
+
+        beforeEach(function() { gd = createGraphDiv(); });
+
+        afterEach(destroyGraphDiv);
+
+        function plot(config) {
+            return Plotly.plot(gd, [], {}, config);
+        }
+
+        it('should fill in scrollZoom default', function(done) {
+            plot(undefined).then(function() {
+                expect(gd._context.scrollZoom).toBe('gl3d+geo+mapbox');
+                expect(gd._context._scrollZoom).toEqual({gl3d: 1, geo: 1, mapbox: 1});
```

Comment:
Could we test `cartesian` to be `undefined` here?

Reply:
Ok, done in -> 09f1af5c3